contribs/gmf/less/desktoplayertree.less

```diff
@@ -11,8 +13,8 @@ gmf-layertree {
 
 .gmf-layertree-node {
 
-  .rightButtons {
-    visibility: hidden;
+  .right-buttons {
+    display: none;
```

Comment:
Because I want the space to be available for the label when the buttons are hidden.